cluster/types.go

```diff
@@ -0,0 +1,93 @@
+// Copyright © 2022 Obol Labs Inc.
```

Comment:
@corverroos it is cleaner to have a params.go operator.go lock.go  and distvalidator.go and then the methods of each in that file

Reply:
I chose the option to colocate logic, almost like how eth2client does things.

Reply:
in eth2client they subsecuently split things of a component into subjects (as they do in prysm) for the same component, but I don't see they do exactly what you do here of putting every component together and then put the same subject of all the components in the same file.

Reply:
ok, grouping files by type. So now a single file for each type

Reply:
Thanks